README.md

```diff
@@ -60,6 +60,9 @@ Please report all issues and feature requests in [cloudfoundry/routing-release](
         "name": "HEALTH_CHECK_NAME",
         "script_path": "/path/to/check/executable",
         "timeout": "HEALTH_CHECK_TIMEOUT"
+      },
+      "options": {
+        "lb_algorithm": "least-connections"
```

Comment:
```suggestion
        "lb_algorithm": "least-connection"
```

We should stick to the values we already have [on the gorouter spec](https://github.com/cloudfoundry/routing-release/blob/develop/jobs/gorouter/spec#L102), namely `round-robin` and `least-connection` (without the 's')

Reply:
Good point. I would keep the same naming for the property - `lb_algo`, mentioned in the [RFC](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0027-generic-per-route-features.md#gorouter).